python/smqtk/utils/image_utils.py

```diff
@@ -0,0 +1,54 @@
+import io
+import PIL.Image
+
+from smqtk.representation.data_element.file_element import DataFileElement
+
+
+def is_loadable_image(dfe):
+    """
+    Determine if an image is able to be loaded by PIL.
+
+    :param dfe: A data file element to check
+    :type dfe: DataFileElement
+
+    :return: Whether or not the image is loadable
+    :rtype bool:
+
+    """
+    try:
+        PIL.Image.open(io.BytesIO(dfe.get_bytes()))
+        return True
+    except IOError, ex:
+        return False
+
+
+def is_valid_element(fp, valid_content_types=None, check_image=False):
+    """
+    Determines if a given filepath is a valid data file element.
+
+    :param fp: File path to element
+    :type fp: str
+
+    :param valid_content_types: List of valid content types, or None to skip
+        content type checking.
+    :type valid_content_types: iterable | None
+
+    :param check_image: Whether or not to try loading the image with PIL. This
+        often catches issues that content type can't, such as corrupt images.
+    :type check_image: bool
+
+    :return: The data file element in the event of a valid element, or None if
+        it's invalid.
+    :rtype: DataFileElement | None
```

Comment:
In isolation, I think it might make more sense if this function returned a boolean.